python/test/hail/methods/test_statgen.py

```diff
@@ -965,14 +965,14 @@ def test_ld_prune_with_duplicate_row_keys(self):
         self.assertEqual(pruned_table.count(), 1)
 
     def test_balding_nichols_model(self):
-        from hail.stats import TruncatedBetaDist
-
         ds = hl.balding_nichols_model(2, 20, 25, 3,
                                       pop_dist=[1.0, 2.0],
                                       fst=[.02, .06],
-                                      af_dist=TruncatedBetaDist(a=0.01, b=2.0, min=0.05, max=0.95),
+                                      af_dist=hl.rand_beta(a=0.01, b=2.0, lower=0.05, upper=0.95, seed=1),
```

Comment:
so determinism is handled by seeding the `af_dist`? If so, you should explain that clearly in the docs

Reply:
Actually, I'm confused how the seed here interacts with the seed parameter below

Reply:
everything else is seeded with the `seed` argument. I'm not really sure what to do here, interface-wise, because I like the idea of passing in the af_dist as a hail function but also the idea of seeding it explicitly but the hail functions already come with a seed

Reply:
and I think having one seed for the `af_dist` and a separate seed for everything else is maybe not the right thing to do but I'm not sure what the right thing to do is

Reply:
I think just be clear in the docs that the `seed` parameter on the method does not determine the seed parameter in `af_dist`; the latter can be left as the default value or set independently, and the resulting dataset is a deterministic function of both seeds.